diff --git a/node_modules/vaul/dist/index.js b/node_modules/vaul/dist/index.js
index cb2693b..1ab8efa 100644
--- a/node_modules/vaul/dist/index.js
+++ b/node_modules/vaul/dist/index.js
@@ -1075,10 +1075,10 @@ function Root({ open: openProp, onOpenChange, children, shouldScaleBackground, o
                 borderRadius: `${BORDER_RADIUS}px`,
                 overflow: 'hidden',
                 ...isVertical(direction) ? {
-                    // transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
+                    transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
                     transformOrigin: 'top'
                 } : {
-                    // transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
+                    transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
                     transformOrigin: 'left'
                 },
                 transitionProperty: 'transform, border-radius',
@@ -1177,10 +1177,10 @@ function Root({ open: openProp, onOpenChange, children, shouldScaleBackground, o
                 borderRadius: `${BORDER_RADIUS}px`,
                 overflow: 'hidden',
                 ...isVertical(direction) ? {
-                    // transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
+                    transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
                     transformOrigin: 'top'
                 } : {
-                    // transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
+                    transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
                     transformOrigin: 'left'
                 },
                 transitionProperty: 'transform, border-radius',
diff --git a/node_modules/vaul/dist/index.mjs b/node_modules/vaul/dist/index.mjs
index e3d2968..4918a0b 100644
--- a/node_modules/vaul/dist/index.mjs
+++ b/node_modules/vaul/dist/index.mjs
@@ -1053,10 +1053,10 @@ function Root({ open: openProp, onOpenChange, children, shouldScaleBackground, o
                 borderRadius: `${BORDER_RADIUS}px`,
                 overflow: 'hidden',
                 ...isVertical(direction) ? {
-                    // transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
+                    transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
                     transformOrigin: 'top'
                 } : {
-                    // transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
+                    transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
                     transformOrigin: 'left'
                 },
                 transitionProperty: 'transform, border-radius',
@@ -1155,10 +1155,10 @@ function Root({ open: openProp, onOpenChange, children, shouldScaleBackground, o
                 borderRadius: `${BORDER_RADIUS}px`,
                 overflow: 'hidden',
                 ...isVertical(direction) ? {
-                    // transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
+                    transform: `scale(${getScale()}) translate3d(0, calc(env(safe-area-inset-top) + 14px), 0)`,
                     transformOrigin: 'top'
                 } : {
-                    // transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
+                    transform: `scale(${getScale()}) translate3d(calc(env(safe-area-inset-top) + 14px), 0, 0)`,
                     transformOrigin: 'left'
                 },
                 transitionProperty: 'transform, border-radius',
